WordPress/Classes/Models/Blog.m

```diff
@@ -400,6 +402,19 @@ - (NSSet *)allowedFileTypes
     return [NSSet setWithArray:allowedFileTypes];
 }
 
+- (void)setOptions:(NSDictionary *)options
+{
+    [self willChangeValueForKey:@"options"];
+    [self setPrimitiveValue:options forKey:@"options"];
+    self.jetpack = nil;
```

Comment:
Since this is non-obvious, a comment explaining why this method is setting `self.jetpack = nil` would be a good addition here.